config/settings-production.yml

```diff
@@ -90,4 +90,18 @@ buffy:
       mapping:
         u: repourl
       template_file: goodpractice.md
+    external_service:
+      name: goodtoreview
+      only:
+        - software-review-testing
+      command: goodtoreview
```

Comment:
other commands have several words but this is fine I suppose

Reply:
can rename later once we confirm that this all works